### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by jburel**
work in progress

### CODE REVIEW

File: data.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+   * http://downloads.openmicroscopy.org/images/DV/U2OS/
+
+
+Data available to trainers only
+-------------------------------
+
+- "INMAC384-DAPI-CM-eGFP_59223_1.xdce" plate:
+   * http://downloads.openmicroscopy.org/images/HCS/INCELL2000/INMAC384-DAPI-CM-eGFP_59223_1/
+- "Brain" dataset:
+   * http://www.osirix-viewer.com/resources/dicom-image-library/ (Alias Name: BRAINIX)
```
Comment (sbesson) [inline data.txt]: Discussed with @jburel, these are no longer publicly available and should probably be replaced by links to DICOM images under our own downloads

Comment (jburel) [reply data.txt]: The Dicom images that we currently have under downloads are not as good.

Comment (jburel) [reply data.txt]: I have found another resource that we could use.

**Comment by jburel** [comment]
@pwalczysko @sbesson happy with the status of this initial branch.
I will review the data section when we are happy with the set of DICOM images

**Comment by pwalczysko** [comment]
See https://github.com/ome/omero-fpbioimage/pull/18#issuecomment-385383425 regarding the "happiness" with the set of DICOM images. Definitely happy if the situation is as on outreach at present, but not happy with the situation on eel.

**Comment by jburel** [comment]
Merging so we can take it from there

**Pull request merged**